ktor-features/ktor-auth-jwt/jvm/src/io/ktor/auth/jwt/JWTAuth.kt

```diff
@@ -25,20 +25,6 @@ private val JWTAuthKey: Any = "JWTAuth"
 
 private val JWTLogger: Logger = LoggerFactory.getLogger("io.ktor.auth.jwt")
 
-/**
```

Comment:
Those couldn't be simply removed. A smooth migration is required. The new type `JWToken` should still have `payload` property so old validation code should keep working.

```
validate { jwt ->
    if (jwt.payload .....) jwt else ... // payload usage should lead to deprecation warning
}
```

The other issue is binary compatibilty. Most like we'll need few steps to migrate

Reply:
Indeed the breakage is not great, and perhaps went too far from the initial goal of QoL changes